src/engraving/libmscore/chordlist.cpp

```diff
@@ -1432,6 +1434,799 @@ qreal ChordList::position(const QStringList& names, ChordTokenClass ctc) const
     }
 }
 
+//---------------------------------------------------------
+//   findModifierStartIndices
+//---------------------------------------------------------
+void ParsedChord::findModifierStackIndices()
+{
+    QStringList alterationsList = { "b", "bb", "#", "##", "natural" };
+    QStringList addOmitList = { "omit", "no", "add" };
+    for (int index = 0; index < _tokenList.size(); index++) {
+        const ChordToken& tok = _tokenList.at(index);
+        if (tok.names.size() > 0) {
```

Comment:
You can slightly simplify this block of code:

if (tock.names.isEmpty()) {
continue;
}

if (alterationsList.contains(tok.names.first())) {
..